app/Providers/AppServiceProvider.php

```diff
@@ -11,7 +23,12 @@ class AppServiceProvider extends ServiceProvider
      */
     public function register(): void
     {
-        //
+        $this->app->bind(UserRepositoryInterface::class,UserRepository::class);
```

Comment:
Mục đích của bind ở đây để làm gì?

Reply:
Means binding an interface UserRepositoryInterface to a specific implementation class UserRepository.
whenever something requests an instance of RepositoryInterface, it should provide an instance of BaseRepository.